### ISSUE TITLE
Should synonym scopes be contextual?

### ISSUE THREAD

**Issue by matentzn** [obophenotype/uberon#2363]
One thing I have never thought about is the fact that synonym scope (exact, broad, related, etc) is not an absolute truth, but depends on the context. 

Consider this 

![image](https://user-images.githubusercontent.com/7070631/160186816-050bba57-8eb0-438d-8095-af77bf09bfb7.png)

Brain stem is considered a synonym (sometimes referred to as...) to hindbrain, while in other sources, such as Uberon this is certainly not the case:

https://www.ebi.ac.uk/ols/ontologies/uberon/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0002028

https://www.ebi.ac.uk/ols/ontologies/uberon/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0002298

**There is a case to be made to capture this kind of nuance.** For example we can have a statement like:

```
"hindbrain" oio:hasExactSynonym "brainstem" {source="brainlab.org"}
```

which case safely co-exist with a statement like:

```
"hindbrain" oio:hasRelatedSynonym "brainstem" {source="uberon"}
```

However, one of the standard QC checks in OBO is looking for cases where the same synonym is mentioned with different scopes, which is not allowed:

http://robot.obolibrary.org/report_queries/duplicate_scoped_synonym

So basically what we need to decide is how we want to treat the "duplicate synonym scope" problem moving forward. This will have implications for all of OBO, so we should do it carefully:

**Should we allow diverging synonym scopes for the same property if (and only if) the source of the synonym is different?**

This implies that if we attach a source or xref to a property, we not only say "the property comes from there" (provenance), but also "in that context this synonym is used to mean an exact synonym to..".

**Comment by cmungall** [comment]
V good points. Here is my quick simplistic picture:

There should be a special global context for a whole ontology, and the exact/narrow etc should always be interpreted in that scope. We need to do a better job of specifying that. For uberon it is pan-metazoan anatomy with a slight human bias, and sometimes with specific biases in terms of different subsystems like the brain, eg a preference for the modern terminology of X vs the older terminology of Y.

I am open to having a mechanism to allow subcontext-specific scope assignments, such that S could be exact for C in the overall context of uberon, but S could be narrow in another context.

**Comment by shawntanzk** [comment]
CL call discussion:

Possible solutions:
- One synonym on a single class with multiple scopes, not allowed unless provenance is attached.  (different provenance for different scopes?)
- One synonym on multiple classes with exact scope - 
     - Allowed but only for abbreviations (must be typed).
- One synonym on multiple classes, with any other scope - allowed
- Abbreviations:
     - Should be tagged as synonym type
     - Allow non-uniqueness.

**Comment by anitacaron** [comment]
We're dropping this issue from the tech board and let the CL editors decide the next steps.

**Comment by anitacaron** [comment]
@bvarner-ebi, could you please discuss the next steps in the next UBERON/CL call? [There're many possible solutions](https://github.com/obophenotype/uberon/issues/2363#issuecomment-1104089640).

**Comment by bvarner-ebi** [comment]
> @bvarner-ebi, could you please discuss the next steps in the next UBERON/CL call? [There're many possible solutions](https://github.com/obophenotype/uberon/issues/2363#issuecomment-1104089640).

Added to agenda.

**Comment by bvarner-ebi** [comment]
Discussed in Uberon call.

There was consensus that the intention of the ontology is not meant to record how a synonym is used in specific sources, but rather for the editor, after doing due diligence in researching the terms/synonyms, to decide how a term is used in the scientific community. It is also uncertain what the benefit would be for end-users to have synonym scopes to be contextual at the granularity of a specific resource. With that said, adding dbxrefs to synonyms is encouraged so that a user can make an informed decision on usage.

Regarding the [example presented](https://github.com/obophenotype/uberon/issues/2363#issue-1181179113), after performing due diligence, an editor would probably determine the reference noted does not reflect the general consensus of neuroanatomy, specifically that "hindbrain" and "brainstem" are not used interchangeably as the brainstem excludes the cerebellum while the hindbrain includes it. This is actually reflected in illustrations on that webpage, even though the text suggests otherwise. Additionally, the information on the site is presented anonymously, which should also give pause to an editor before incorporating it into a reference ontology.

It was also discussed that for synonyms with type = abbreviation, these should be added as related and not exact synonyms. Adding them as exact by default is not in line with the scope definitions and can be problematic in cases where NLP is applied.

> Possible solutions:
> - One synonym on a single class with multiple scopes, not allowed unless provenance is attached.  (different provenance for different scopes?)

There is NOT agreement on the first point.

> - One synonym on multiple classes with exact scope - 
>      - Allowed but only for abbreviations (must be typed).

This should ONLY be allowed for abbreviations, which should be RELATED synonyms.

> - One synonym on multiple classes, with any other scope - allowed

Agreed that this is acceptable.

> - Abbreviations:
>      - Should be tagged as synonym type
>      - Allow non-uniqueness.

Agreed, and they should have scope: RELATED.

**Comment by matentzn** [comment]
- [ ] Ok, I think this can be closed if this consensus is captured in a 6 bullet or so Markdown file, added to the Uberon docs.

**Comment by bvarner-ebi** [comment]
> [x] Ok, I think this can be closed if this consensus is captured in a 6 bullet or so Markdown file, added to the Uberon docs.

Addressed in PR #2966. 
Thanks, all, for the feedback and review.
